1-exercises/A-this/exercise-1.js

```diff
@@ -9,18 +9,17 @@
 
 */
 
-
-
 let person = {
-    name: "Adrian",
-    age: 32,
-    currentAddress: "London",
-    celebrateBirthday: /* todo */
+  name: "Adrian",
+  age: 32,
+  currentAddress: "London",
+  celebrateBirthday: function () {
+    return this.age;
```

Comment:
This celebrateBirthday function should increment the age of the person by 1 & return the current age